Accessibility updates to codelab.md, accessibility.md (#7678)
flutter/website: src/development/accessibility-and-localization/accessibility.md

<iframe width="560" height="315" src="{{site.youtube-site}}/embed/bWbBgbmAdQs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

## Testing accessibility on mobile

> You can use Flutter's <a href="https://api.flutter.dev/flutter/flutter_test/AccessibilityGuideline-class.html">Accessibility Guideline API</a> to test if your mobile app's UI meets the accessibility recommendations for text contrast, target size, and target labels. 

Reviewer: **issue:** Update copy with semantic line breaks and more concise copy.
```suggestion
Test your app using Flutter's
<a href="https://api.flutter.dev/flutter/flutter_test/AccessibilityGuideline-class.html">Accessibility Guideline API</a>.
This API checks if your app's UI meets Flutter's accessibility recommendations.
These cover recommendations for text contrast, target size, and target labels.
```

> The example below shows how to use the Guideline API on <a href="https://docs.flutter.dev/get-started/codelab/?tab=talkback">Startup Name Generator</a>, the app built as part of the codelab on creating your first Flutter app. Each list tile on the app's main screen is tappable target with text represented in 18 point. You can add the tests provided by the Guideline API in `test/widget_test.dart` of your app directory.

Reviewer: **issues:** 
- Replace "below" as it runs counter to [Google's documentation accessibility guidelines](https://developers.google.com/style/accessibility).
- Use [semantic line breaks](https://sembr.org/) for copy.

```suggestion
The following example shows how to use the Guideline API on
<a href="https://docs.flutter.dev/get-started/codelab/?tab=talkback">Startup Name Generator</a>,
the app built as part of the codelab on creating your first Flutter app.
Each list tile on the app's main screen serves as a tappable target
with text represented in 18 point. You can add Guideline API tests
in `test/widget_test.dart` of your app directory.
```

<?code-excerpt path-base="codelabs/startup_namer/step3_stateful_widget"?>

> <?code-excerpt "test/widget_test.dart" title indent-by="2"?>

Reviewer 2: It looks like this code excerpt is out of date. 

When I run [`./tool/refresh-code-excerpts.sh`](examples/codelabs/startup_namer/step3_stateful_widget/test/widget_test.dart), it replaces the entire snippet with the contents from the file at : `examples/codelabs/startup_namer/step3_stateful_widget/test/widget_test.dart`

Mauli, you'll have to integrate that excerpt into that widget_test.dart file in the codelabs repo (cc @domesticmouse), land it, and then pull in the updated codelab submodule into this PR. The refresh code excerpts should be happy when that's completed.

Reviewer 3: The updated codelab that should land in the next update cycle will contain @pandeymauli's updated `widget_test.dart`, so hopefully everything just works?
